corio.py

```diff
@@ -110,6 +110,8 @@ def parse_args():
     parser.add_argument("-dm", "--degraded_mode", type=lambda x: bool(strtobool(str(x))),
                         default=False,
                         help="Degraded Mode, True/False")
+    parser.add_argument("-mr", "--s3max_retry", type=int, default=1,
```

Comment:
Default can be 0 or 6

Reply:
Default should be 0. Let user decide on max retry.